cli.js

```diff
@@ -25,10 +29,8 @@ yargs
     global: true,
 })
 
-.command('observe', 'Starts observing module changes and pushes them into the queue', require('./cli/observe'))
-.command('consume', 'Consumes modules from the queue, analyzing them', require('./cli/consume'))
-.command('scoring', 'Continuously iterate over the analyzed modules, scoring them', require('./cli/scoring'))
-
-.command('tasks', 'Various useful tasks', require('./cli/tasks'))
-
+.command('observe', 'Starts observing module changes and pushes them into the queue', observe)
```

Comment:
reverted! (although I do think that having all the requires at the top of the file makes it easier to read)